cmd/root.go

```diff
@@ -1,19 +1,31 @@
 package cmd
 
 import (
+	"bufio"
+	"context"
 	"encoding/json"
 	"fmt"
+	"io/fs"
 	"log"
 	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/Xuanwo/go-locale"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/thegreenwebfoundation/grid-intensity-go/ember"
+	"github.com/thegreenwebfoundation/grid-intensity-go/watttime"
```

Comment:
Just a style thing. I always find the 3 T's in a row hard to read. you reckon it might be worth having this as `watt-time` in the code, to match the WATT_TIME_FOO environment variables elsewhere?

Reply:
Yes, the triple T is hard to read. Unfortunately hyphens aren't recommended for go package names.
https://go.dev/blog/package-names

I'll have a deeper look but I think we may have to live with the current name.